serializer/serializer.c

```diff
@@ -550,6 +550,17 @@ status_t send_mam_req_deserialize(const char* const obj, ta_send_mam_req_t* req)
     goto done;
   }
 
+  json_result = cJSON_GetObjectItemCaseSensitive(json_obj, "prng");
```

Comment:
how about using 
```
if (cJSON_HasObjectItem(json_obj, "bundles")) {
    ret = json_array_to_hash243_queue(json_obj, "bundles", &out->bundles);
    if (ret) {
      goto end;
    }
}
...
```
link https://github.com/iotaledger/entangled/blob/develop/cclient/serialization/json/find_transactions.c#L82

Reply:
This is a tryte string.

Reply:
I mean that we could use `cJSON_HasObjectItem(json_obj, "bundles")` to determine if the key existes

Reply:
Did previous cleanup also handle this?

Reply:
cleanup in TA? I haven't done this

Reply:
It also requires another function to get the string and it's still `cJSON_GetObjectItemCaseSensitive` needed. I prefer to stay in this way.

Reply:
Sure